lib/Cargo.toml

```diff
@@ -57,6 +57,8 @@ pulldown-cmark = { version = "0.11.0", optional = true }
 calamine = { version = "0.24.0", optional = true }
 rust_xlsxwriter = { version = "0.64.2", optional = true }
 shiva-spreadsheet-ods = { version = "0.0.2", optional = true }
+strum = { version = "0.26", features = ["derive"] }
+wasm-bindgen = "0.2.84"
```

Comment:
Could you remove unused dependency?

Reply:
Fixed the error. PDF was not in Cargo.toml of shiva-wasm. About strum, I'm  using it to empower Enum, see the use in CLI with snap:
`#[arg(long, value_parser = DocumentType::variants_as_str() )]
    input_format: String,
`
About the wasm-bindgen, I didn't like it either but as I look other projects, and since DocumentType will be used as wasm_bindgen, I have to use it, otherwise e just return the enum FileFomat on shiva-wasm.

Reply:
Well, this problem isn't in my code at all; it happens when we enable the pdf feature in the Cargo configuration for shiva-wasm. I tried to fix it, but the ureq dependency isn't supposed to be used in a WebAssembly (wasm) environment (as it involves making requests inside wasm). I did some research and tested a code with a new library, ehttp, and it worked. However, for some reason, the generated wasm file now has a size of 26 MB, which I think is too large.

Do we need the PDF in wasm right? my suggestion for now is to comment the ureq code in typst.rs

Reply:
Do you a test please in your main branch. Change your Cargo feature on shiva-wasm and try to build. You will see the ureq problem.

Reply:
We know about ureq problem with wasm. This main reason why pdf and typst disabled in "shiva-wasm".
But you send pull request befor solving that problem. You have to choose:
1. fix that problem by self and I will merge your pull request after that
2. wait fix from us and only after that I will merge your pull request in future

Reply:
Fixed! But, tbh, now with PDF and typst, WASM goes brr...